extensions/hadoop-dist/pom.xml

```diff
@@ -127,6 +127,12 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>ch.qos.reload4j</groupId>
+                    <artifactId>reload4j</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
If the dependency is not needed it should be excluded here:

https://github.com/hazelcast/hazelcast/blob/master/pom.xml#L1557-L1562

I also think we can remove log4j from there because reload4j is a replacement.

Reply:
I've decided to leave the exclusion here, as we need reload4j in tests